dss/operations/secrets.py

```diff
@@ -362,13 +362,16 @@ def __init__(self, stage):
         self.stage_env = self.get_stage_env(self.stages[self.stage])
         self.service_account = self.fetch_terraform_output("service_account", "gcp_service_account").strip()
 
-        self.email = [f'{self.service_account}@human-cell-atlas-travis-test.iam.gserviceaccount.com']
-        self.project = ['human-cell-atlas-travis-test']
+        # the value `platform-hca` is used here rather than `platform-sc` due to other developers wanting
```

Comment:
Good thinking, this project name/ID thing is going to get confusing with the fog of time